dotcom-rendering/src/footballTables.ts

```diff
@@ -1,9 +1,12 @@
-import type { TeamScore } from './footballMatches';
+export type TeamResultScore = {
+	name: string;
+	score: number;
```

Comment:
Score is optional in `TeamScore` (because sometimes there's no score from PA for a live game)